sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala

```diff
@@ -192,7 +137,25 @@ class HiveScriptTransformationSuite extends SparkPlanTest with SQLTestUtils with
     assert(uncaughtExceptionHandler.exception.isEmpty)
   }
 
-  test("SPARK-25990: TRANSFORM should handle different data types correctly") {
+  test("SPARK-30973: TRANSFORM should wait for the termination of the script (with serde)") {
```

Comment:
how about `with serde` -> `hive serde` in the existing tests names?

Reply:
> how about `with serde` -> `hive serde` in the existing tests names?

all right, better.